docs/api-docs/headless/carts.md

```diff
@@ -89,20 +118,25 @@ The response will contain `cart_url` and `checkout_url` parameters - use these U
 
 It is possible to generate a redirect URL when creating a cart using the [Create a Cart](/api-reference/store-management/carts/cart/createacart) endpoint by appending the `include=redirect_urls` query parameter to the request URL.
 
-```http
-POST https://api.bigcommerce.com/stores/{store_hash}/v3/carts?include=redirect_urls
+```http title="Create a cart"
+POST https://api.bigcommerce.com/stores/{{store_hash}}/v3/carts?include=redirect_urls
 Accept: application/json
 Content-Type: application/json
 X-Auth-Token: {{ACCESS_TOKEN}}
+
+{
+  // request body
+}
+
 ```
 
-### Logging in and redirecting a customer
+### Signing in and redirecting a customer
 
```

Comment:
@bc-traciporter Let's revise this section to point to our dedicated documentation on the Customer Login API.